### PR TITLE
Add generic parameters to builder methods

### PR DESCRIPTION
**Pull request by antoyo**
[Example of new code generation.](https://github.com/gtk-rs/gtk/pull/902/files#diff-b636ac4109f36a9dcd7198cf7ed90654L548)

### CODE REVIEW

File: src/analysis/class_builder.rs [hunk 1]
```diff
@@ -98,14 +99,21 @@ fn analyze_property(
     if !for_builder {
         return None;
     }
-    if let Ok(ref s) = used_rust_type(env, prop.typ, false) {
+    let type_str = used_rust_type(env, prop.typ, false);
+    if let Ok(ref s) = type_str {
         if !s.contains("GString") {
             imports.add_used_type_with_version(s, prop_version);
```
Comment (EPashkin) [inline src/analysis/class_builder.rs]: Please add [`set_defaults`](https://github.com/gtk-rs/gir/blob/master/src/analysis/properties.rs#L111) before this if, and just `add_used_type`, then `imports.add("glib::object::IsA");` will have same version

**Comment by EPashkin** [comment]
@antoyo LGFM, Thanks. Only one small nit

**Comment by antoyo** [comment]
@EPashkin: Please tell me if I did the correct change in the last commit.

**Comment by EPashkin** [comment]
@antoyo Yes, thanks. Just waiting CI

**Pull request merged**